.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -33,10 +33,19 @@ body:
       description: If your bug produces a backtrace, please include it here.
       render: shell
   - type: textarea
-    id: logs
+    id: layer-logs
     attributes:
-      label: Relevant Logs
-      description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs
+      label: mirrord layer logs
+      description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs.
```

Comment:
typo in environment

Reply:
I'm using archaic english :eyes: 


... fixed